bitnami/mysql/Chart.yaml

```diff
@@ -27,4 +27,4 @@ maintainers:
 name: mysql
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/mysql
-version: 9.10.10
+version: 9.10.11
```

Comment:
As this PR adds new features, could you please bump a minor version (9.11.0)?

Reply:
Done